nucypher/blockchain/eth/agents.py

```diff
@@ -68,6 +70,7 @@
     StakerInfo,
     PeriodDelta,
     StakingEscrowParameters,
+    Policy
```

Comment:
I like that `ArrangementInfo` is defined in PolicyManagerAgent, and find it a little strange that we have some other datatypes that are specific to contracts declared somewhere other than in the respective agent eg. can `Policy` be declared in PolicyAgent instead of in `nucypher/types.py`; can `StakingEscrowParameters` be declared in StakingEscrowAgent instead of in `nucypher/types.py`...?

May not apply to all classes defined in `nucypher/types.py`, but perhaps some/most?

Reply:
Also, in keeping with the naming of other similar classes - wdyt about `PolicyInfo` instead of `Policy`?

Reply:
I am with Derek here regarding `PolicyInfo` vs `Policy`. We already have a `Policy` (defined in `policies.py`), which is a different type (with more complexity too, not just a collection of simple values). Makes sense to follow the example of `Arrangement` (a Python class) vs `ArrangementInfo` (a mapping of a Solidity structure).

Reply:
Renamed `Policy` -> `PpolicyInfo`

Reply:
> Do you have a preference for consistency? If it's the namepacing that matters most to you perhaps we make a typing module per contract?

I think some sort of more direct association between the types defined and the contracts would make them more intuitive. Having them live in `nucypher/types.py` just seems vague; it could be a typing module per contract or even keeping it general like `nucypher.blockchain.eth.types`...?

Reply:
^ BTW Doesn't have to be this PR - maybe the namespacing can be looked into as part of a separate PR.

Reply:
Filed #2739 .